src/genesysmod_equ.jl

```diff
@@ -430,7 +430,7 @@ function genesysmod_equ(model,Sets,Params, Vars,Emp_Sets,Settings,Switch, Maps)
           if 𝓨[i] == Switch.StartYear
             @constraint(model, Vars.TotalTradeCapacity[𝓨[i],f,r,rr] == Params.TradeCapacity[r,rr,f,𝓨[i]], base_name="TrC2a_TotalTradeCapacityStartYear|$(𝓨[i])|$(f)|$(r)|$(rr)")
           elseif 𝓨[i] > Switch.StartYear
-            @constraint(model, Vars.TotalTradeCapacity[𝓨[i],f,r,rr] == Vars.TotalTradeCapacity[𝓨[i-1],f,r,rr] + Vars.NewTradeCapacity[𝓨[i],f,r,rr] + Params.CommissionedTradeCapacity[r,rr,f,𝓨[i]], 
```

Comment:
What is the reason for removing CommissionedTradeCapacity? I think it might not be used now but is still useful to keep

Reply:
I will complete this pull request now, and we can remove CommissionedTradeCapacity in a later one if we decide that it is not going to be used in the future.

Reply:
I did not find it in the input data which is why I removed it due to redundancy but we can of course keep it.